test/acceptance/behave/components/ngsiv2/batch_operations/batch_op_update/batch_op_update_action_type.feature

```diff
@@ -671,7 +671,36 @@ Feature: actionType in update batch operation using NGSI v2. "POST" - /v2/op/upd
       | entities_id        | room2    |
       | attributes_name    | pressure |
     When update entities in a single batch operation "DELETE"
+    Then verify that receive an "Not Found" http code
```

Comment:
I think here is better a 400 code rather than not found. In fact strictly, speaking an empty entity array should not cause any error, but for the sake of developer usability it is ok to send an error